src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -146,7 +146,33 @@ public IList<ITelemetryInitializer> TelemetryInitializers
         {
             get { return this.telemetryInitializers; }
         }
-        
+
+        /// <summary>
+        /// Gets or sets the first telemetry processor in the chain of processors.
+        /// </summary>
+        public TelemetryProcessorChain TelemetryProcessorChain
```

Comment:
done - though i am not sure yet if the correction to th ConfigurationFactory's parsing logic is appropriate.